pkg/execution/executor/service.go

```diff
@@ -212,7 +212,7 @@ func (s *svc) Run(ctx context.Context) error {
 		}
 
 		if err != nil && err.Error() != "NonRetriableError" {
-			s.log.Error("error handling queue item", "error", err, "item", item)
```

Comment:
why remove the item from the log?
it helps with debugging purposes and I don't think it contains PII.

Reply:
it's extremely noisy in the dev server CLI, and contains lots of details that our users do not need to see.  given this runs in the dev server and will primarily be seen by our users... if we want the run ID, we should log the run ID.